examples/solverdummy/solverdummy.f03

```diff
@@ -47,15 +76,28 @@ PROGRAM main
 
   CALL precicef_is_coupling_ongoing(ongoing)
   DO WHILE (ongoing.NE.0)
-  
-    CALL precicef_action_required(writeItCheckp, bool, 50)
+
+    CALL precicef_is_action_required(writeItCheckp, bool, 50)
     IF (bool.EQ.1) THEN
       WRITE (*,*) 'DUMMY: Writing iteration checkpoint'
       CALL precicef_mark_action_fulfilled(writeItCheckp, 50)
     ENDIF
 
-    CALL precicef_advance(dtlimit)
-    CALL precicef_is_coupling_ongoing(ongoing)
+    CALL precicef_is_read_data_available(bool)
+    IF (bool.EQ.1) THEN
+      CALL precicef_read_bvdata(readDataID, numberOfVertices, vertexIDs, readData)
+    ENDIF
+
+    WRITE (*,*) 'readData: ', readData
+
```

Comment:
```suggestion
```

For sake of consistency with https://github.com/precice/precice/pull/750 remove these lines.

Reply:
This was the previous state of https://github.com/precice/precice/pull/750, good that it was removed now.